### ISSUE TITLE
Support for K8s Metrics API ?

### ISSUE THREAD

**Issue by Al-assad** [coralogix/zio-k8s#421]
Hi, Does zio-k8s support the kubernetes metrics api on https://kubernetes.io/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/#metrics-api ?

**Comment by vigoo** [comment]
No, it does not seem to be part of the K8s (v1.22.1) open API definition that the library is generated from:
https://raw.githubusercontent.com/kubernetes/kubernetes/v1.22.1/api/openapi-spec/swagger.json
